pkg/cluster/provider.go

```diff
@@ -105,6 +106,27 @@ func (p *Provider) Delete(name, explicitKubeconfigPath string) error {
 	return internaldelete.Cluster(p.logger, p.ic(name), explicitKubeconfigPath)
 }
 
+// Deletes all clusters created by getting list of cluster names and using Delete function.
+func (p *Provider) DeleteAll(explicitKubeconfigPath string) error {
+	clusters, err := p.provider.ListClusters()
+	if err != nil {
+		return err
+	}
+
+	if len(clusters) == 0 {
+		return errors.New("no kind clusters found")
+	}
+
+	for _, cluster := range clusters {
+		err = internaldelete.Cluster(p.logger, p.ic(cluster), explicitKubeconfigPath)
+		if err != nil {
+			return errors.Wrap(err, "failed to delete cluster "+cluster)
```

Comment:
sorry, this should have been:
`errors.Wrapf(err, "failed to delete cluster %q", cluster)`

Reply:
Ah, with quotes. Sure thing!